### PR TITLE
DR-857 - New fields pages

### PR DESCRIPTION
**Pull request by mvillaseco**
Changes:
 - Remove Domain from Registration.
 - Add Company to registration.
 - Fix tests.
 - Add responsive behaviour.

### CODE REVIEW

File: src/wwwroot/partials/signup/registration.html [hunk 1]
```diff
@@ -46,8 +46,16 @@
         <span class="input--error" ng-show="(vm.submitted || form.accountName.$dirty) && form.accountName.$error.accountname_already_taken"><i class="arrow--up"></i>{{'validation_error_accountname_already_taken' | translate}}</span>
       </div>
     </div>
-  </form>
   <div class="flex flex--center section--wrapper-input">
-    <button class="button button--medium" type="submit" ng-click="vm.submitRegistration(form)">{{'registration_submit' | translate}}</button>
+    <div class="wrapper-input width--full">
+      <input class="check" ng-model="checkbox" name="checkbox" type="checkbox" /><span>{{'confirmation_terms' | translate}}</span><a target="_blank" href="https://www.dopplerrelay.com/privacidad">{{'confirmation_terms_link' | translate}}</a><a target="_blank" href="https://www.dopplerrelay.com/terminos-condiciones">{{'confirmation_terms_2' | translate}}</a>
```
Comment (mvillaseco) [inline src/wwwroot/partials/signup/registration.html]: Remove "check" class that it's not being used.

File: src/wwwroot/controllers/RegistrationCtrl.js [hunk 2]
```diff
@@ -33,7 +32,7 @@
         lastName: vm.lastName,
         password: vm.password,
         account_name: vm.accountName,
```
Comment (andresmoschini) [inline src/wwwroot/controllers/RegistrationCtrl.js]: Could you add a validation for accountName? it should only accept lowecase letters, numbers, `_` and `-`.

File: src/wwwroot/partials/signup/registration.html (hunk 1 above)
Comment (andresmoschini) [inline src/wwwroot/partials/signup/registration.html]: Remember to send this value to API

File: src/wwwroot/partials/signup/confirmationPage.html [hunk 3]
```diff
@@ -15,26 +15,45 @@
...
+          <span class="input--error" ng-show="(vm.submitted || form.confpass.$dirty) && form.confpass.$error.same"><i class="arrow--up"></i>{{'login_error_same_password' | translate}}</span>
+        </div>
       </div>
       <div class="confirmation--submit-container">
-        <div class="wrapper-input">
-          <input ng-model="checkbox" name="checkbox" type="checkbox" required /><span>{{'confirmation_terms' | translate}}</span><a target="_blank" href="https://www.dopplerrelay.com/privacidad">{{'confirmation_terms_link' | translate}}</a><a target="_blank" href="https://www.dopplerrelay.com/terminos-condiciones">{{'confirmation_terms_2' | translate}}</a>
+        <div class="wrapper-input" ng-if="vm.termsAccepted">
+          <input ng-model="checkTerms" name="checkTerms" type="checkbox" value="term1" required /><span>{{'confirmation_terms' | translate}}</span><a target="_blank" href="https://www.dopplerrelay.com/privacidad">{{'confirmation_terms_link' | translate}}</a><a target="_blank" href="https://www.dopplerrelay.com/terminos-condiciones">{{'confirmation_terms_2' | translate}}</a>
```
Comment (andresmoschini) [inline src/wwwroot/partials/signup/confirmationPage.html]: Could you move the value constant to controller?

Comment (andresmoschini) [inline src/wwwroot/partials/signup/confirmationPage.html]: Also, could you set it as 1? in place of term1?

File: src/wwwroot/controllers/ConfirmationCtrl.js [hunk 4]
```diff
@@ -46,6 +49,8 @@
           vm.userName = result.data.user_email;
           vm.name = result.data.firstName || result.data.user_email;
           vm.domain = result.data.domain;
+          vm.emptyPass = result.data.emptyPass;
```
Comment (andresmoschini) [inline src/wwwroot/controllers/ConfirmationCtrl.js]: what do you think about `password_empty`?

File: src/wwwroot/controllers/ConfirmationCtrl.js [hunk 5]
```diff
@@ -46,6 +49,8 @@
...
+          vm.termsAccepted = result.data.termsAccepted;
```
Comment (andresmoschini) [inline src/wwwroot/controllers/ConfirmationCtrl.js]: what do you think about `terms_conditions` or `accepted_terms_conditions`?

Comment (andresmoschini) [inline src/wwwroot/controllers/ConfirmationCtrl.js]: Or `terms_and_conditions_version`

File: src/wwwroot/services/signup.js [hunk 6]
```diff
@@ -105,7 +105,11 @@
         },
         data: {
           "domain": domain,
-          "password": pass
+          "password": pass,
+          "industry": industry,
```
Comment (andresmoschini) [inline src/wwwroot/services/signup.js]: What do you think about industry_code?

File: src/wwwroot/services/signup.js [hunk 7]
```diff
@@ -105,7 +105,11 @@
...
+          "phone_number": phoneNumber,
+          "country": country,
```
Comment (andresmoschini) [inline src/wwwroot/services/signup.js]: What do you think about country_code?

File: src/wwwroot/services/signup.js [hunk 8]
```diff
@@ -105,7 +105,11 @@
...
+          "terms": termsAndRights
```
Comment (andresmoschini) [inline src/wwwroot/services/signup.js]: What do you think about `terms_and_conditions_version`?

File: src/wwwroot/services/signup.js [hunk 9]
```diff
@@ -133,7 +137,8 @@
           'lastName': newUser.lastName,
           'password': newUser.password || null,
           'account_name': newUser.account_name,
-          'domain': newUser.domain || null
+          'company': newUser.company || null,
```
Comment (andresmoschini) [inline src/wwwroot/services/signup.js]: What do you think about company_name?

File: src/wwwroot/services/signup.js (hunk 8 above)
Comment (andresmoschini) [inline src/wwwroot/services/signup.js]: Missed telephone field

**Pull request merged**